packages/serverless/src/domain/use-case/group.ts

```diff
@@ -22,15 +21,15 @@ export class Group {
   /**
    * Checks if a group exists based on the provided parameters.
    *
-   * @param params - The parameters used to filter and identify the group, following the `TGroupParamBase` type.
+   * @param paramGroupExist - The parameters used to filter and identify the group, following the `TGroupParamBase` type.
    * @param session - (Optional) The MongoDB session for transactional queries.
    * @returns A promise resolving to `true` if a matching group exists, otherwise `false`.
    */
   public static async exist(
-    params: TGroupParamExist,
+    paramGroupExist: TGroupParamExist,
```

Comment:
Shouldn't define too may of param kinds, just the common one.

Reply:
I just refactor. remove redundant define